README.md

```diff
@@ -22,51 +22,114 @@ All of these actions require us to use **programmatic navigation** to change the
 browser URL, and show the user a new page in our application, **without** making
 the user click on a link.
 
-**Note:** We have attached some files so you can see an example app in which
-programmatic navigation has been implemented. While the basic functionality is
-the same as what's described below, there are a number of differences for two
-reasons: 1) the code in the lesson is substantially pared down to make it easier
-to focus on the specific functionality being explained; and 2) the login/logout
-functionality is mocked in the example app so you don't need to run a server. We
-recommend that you read through the lesson first, focusing on understanding how
-programmatic navigation works. Once you've done that, feel free to start up the
-app and explore a (mostly) working example.
+We've included some files for you to code along in as we walk through the
+examples below. The files already have routing set up - we just need to update
+them to include programmitic navigation!
 
-## The useHistory Hook
+## The useNavigate Hook
 
-To solve this problem, we can use another custom hook from React Router: the
-`useHistory` hook. Here's how it looks:
+To enable programmatic navigation, we can use another custom hook from React
+Router: the `useNavigate` hook. Here's how we could include it in our
+application.
+
+First, we need to import it into the component in which we want to use it:
+`import { useNavigate } from 'react-router-dom'`. In this case, we'll be
+importing into our `App` component.
+
+Then, we need to invoke our `useNavigate` hook and save the returned function in
+a variable. Let's call that variable `navigate` for simplicity: `const navigate
```

Comment:
I'm fine with removing the "Let's call that variable `navigate` etc." portion, but is there a reason we want to get rid of the `const navigate = useNavigate()` portion? I included that and the other short code examples here because the full example down below is a little busy. 

I thought it might be easier for students to pick out the key pieces of functionality being demonstrated if they had some very simple examples to look at first. What do you think?

Reply:
I do see that you've added comments to the examples, which I think is a great addition, but I think having both could be helpful for students.

Reply:
The reason I did that is because when I was reading it, the amount of detail made me think I was supposed to be making the updates to my code at that point. I also wanted to add comments to the code block because it was always frustrating to me as a student when there was a big code block that I'd have to comb through it to find the changes, so I figured a quick bulleted list of the general steps followed by the commented code was a good middle ground. 

Having said that, though, I've never really found a good way to handle this situation and I'm not married to this approach. I'm fine with adding more details to the bulleted steps as long as it's reasonably clear that they don't need to be implementing them yet.